zenodo/modules/spam/views.py

```diff
@@ -126,17 +127,38 @@ def delete(user_id):
         return render_template('zenodo_spam/delete.html', **ctx)
 
 
-def _evaluate_user_domain(email):
-    email_domain = email.rsplit('@', 1)[-1].lower()
+def normalize_email(email):
+    email = email.lower()
+    username = email[: email.index("@")]
+    no_dots_username = username.replace(".", "")
+    domain = email[email.index("@") :]
+
+    if(domain == "@googlemail.com"):
```

Comment:
```suggestion
    if domain == "@googlemail.com":
```

No need for parentheses here :)